scipy/stats/tests/test_generation/reference_distributions.py

```diff
@@ -279,6 +279,21 @@ def _pdf(self, x, a):
     # that there is no *mistake* in the implementation (e.g. wrong formula).
 
 
+class BetaPrime(ReferenceDistribution):
+
+    def __init__(self, *, a, b):
+        super().__init__(a=a, b=b)
+
+    def _logpdf(self, x, a, b):
+        return (a - 1.0)*mp.log(x) - (a + b)*mp.log1p(x) - mp.log(mp.beta(a, b))
```

Comment:
Don't think this will lose precision as it is because `a` is an `mpf`, but it's not a bad habit to use `mp.one` (here and below).
```suggestion
        return (a - mp.one)*mp.log(x) - (a + b)*mp.log1p(x) - mp.log(mp.beta(a, b))
```

Reply:
This comment applies in `_sf`, too, but I won't let that hold things up. Please keep it in mind in the future.